src/index.html

```diff
@@ -4,9 +4,73 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Calendar</title>
-    <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="stylesheet" href="style.css">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <main class="main-content">
+      <section class="calendar">
```

Comment:
These tasks are not implemented 
<img width="804" alt="image" src="https://user-images.githubusercontent.com/99403198/223101652-29bfab7b-0219-49a0-ac95-b9d15a336cda.png">

Reply:
Implement the whole task, please 
![image](https://github.com/mate-academy/layout_calendar/assets/99403198/3aeed0fc-c9b5-4668-bfb9-9721b0f4df5f)